msal_extensions/persistence.py

```diff
@@ -150,8 +153,13 @@ def save(self, content):
 
     def load(self):
```

Comment:
Given that we now head to the direction of using `None` as an indication of ITEM_NOT_FOUND, we better make such intention explicit.
```suggestion
    def load(self):
        """Return content from OSX Keychain, or None if the content does not exist."""
```

Reply:
Added in [this](https://github.com/AzureAD/microsoft-authentication-extensions-for-python/pull/65/commits/c4a9938c2ce187bea8767fee3f05b7d8680f9010) commit